### PR TITLE
Change download URL

### PR DESCRIPTION
**Pull request by traci-morrison**
### Proposed changes
- https://docs.docker.com/ee/docker-ee/ubuntu/
- Changed URL for .deb file download
- Update Note format

### Related issues (optional)
This is a continuation of the work started in https://github.com/docker/docker.github.io/pull/9286.

### CODE REVIEW

**Comment by traci-morrison** [comment]
@Dawn-Wood @KoryKessel-Docker PTAL

File: ee/docker-ee/ubuntu.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Instructions for installing Docker Engine - Enterprise on Ubuntu
+description: Instructions for installing Docker Enterprise on Ubuntu
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: The Title for the page should be changed to reflect the revised description ("Installing Docker Enterprise on Ubuntu").

File: ee/docker-ee/ubuntu.md [hunk 2]
```diff
@@ -156,9 +156,11 @@ and update Docker Engine - Enterprise from the repository.
 6.  Use the following command to set up the **stable** repository. Use the
     command as-is. It works because of the variable you set earlier.
 
-    > **Note:** The `lsb_release -cs` sub-command below returns the name of your
+    > **Note** 
+    > 
+    > The `lsb_release -cs` sub-command below returns the name of your
     > Ubuntu distribution, such as `xenial`.
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: In my opinion, the wording should be slightly changed "The following `lsb_release l-cs` sub command returns the name of your Ubuntu distribution (e.g., `xenial`).

Also, the command that follows should be a part of the note block.

File: ee/docker-ee/ubuntu.md [hunk 3]
```diff
@@ -119,7 +117,9 @@ and update Docker Engine - Enterprise from the repository.
 
 4. Temporarily add a `$DOCKER_EE_VERSION` variable into your environment.
 
-   > **Note:** If you need to run Docker Enterprise 2.0, please see the
+   > **Note** 
+   > 
+   > If you need to run Docker Enterprise 2.0, please see the
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: In my opinion, this note block should come after the CLI example.

File: ee/docker-ee/ubuntu.md [hunk 4]
```diff
@@ -156,9 +156,11 @@ and update Docker Engine - Enterprise from the repository.
 6.  Use the following command to set up the **stable** repository. Use the
     command as-is. It works because of the variable you set earlier.
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Change the wording to read: "Set up the **stable** repository, using the following command as-is (which works due to the variable set up earlier in the process)."

File: ee/docker-ee/ubuntu.md (hunk 2 above)
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Guessing that this note is the result of a cutting-pasting short cut? :-)

**Comment by traci-morrison** [comment]
> Comments within.
> 
> One small thing I could not comment on is the use of 'OK' in the text as a word (a pet peeve I have). The sentence I am referring to reads, "It’s OK if apt-get reports that none of these packages are installed." I think that this should be a note instead that reads: "The apt-get command may report that none of the packages are installed."

Thanks @KoryKessel-Docker. I made your suggested changes. PTAL.

File: ee/docker-ee/ubuntu.md [hunk 5]
```diff
@@ -23,42 +23,41 @@ Docker Engine - Community users should go to
...
 
-> **Note:**
-> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.06.xx only.
+> **Note**
+> 
+> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.04.xx only.
 
 ### Uninstall old versions
 
 Older versions of Docker were called `docker` or `docker-engine`. In addition,
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Change the first sentence to read:, "Use the `sudo apt-get remove` command to uninstall older versions of Docker Engine - Enterprise (called `docker` or `docker-engine`).

File: ee/docker-ee/ubuntu.md [hunk 6]
```diff
@@ -23,42 +23,41 @@ Docker Engine - Community users should go to
...
 
 ```bash
 $ sudo apt-get remove docker docker-engine docker-ce docker-ce-cli docker.io
 ```
 
-It's OK if `apt-get` reports that none of these packages are installed.
+The `apt-get` command may report that none of the packages are installed.
 
 The contents of `/var/lib/docker/`, including images, containers, volumes, and
 networks, are preserved. The Docker Engine - Enterprise package is now called
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: In my opinion, this sentence works better as a Note.

Comment (KoryKessel-Docker) [reply ee/docker-ee/ubuntu.md]: For the section below, change the last two sentences to read: "If you need to use aufs instead, be aware that it must be manually configured (refer to aufs)."

File: ee/docker-ee/ubuntu.md [hunk 7]
```diff
@@ -80,7 +79,6 @@ needs.
   [set up Docker's repositories](#install-using-the-repository) and install
   from them, for ease of installation and upgrade tasks. This is the
   recommended approach.
-
 - Some users download the DEB package and install it manually and manage
   upgrades completely manually. This is useful in situations such as installing
   Docker on air-gapped systems with no access to the internet.
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Under the **Install Docker Engine - Enterprise** heading and before the **Install using the repository** subheading, change all of the text as follows:

Docker Engine - Enterprise can be installed either via Docker repositories, or by downloading and installing the DEB package and thereafter manually managing all upgrades. The Docker repository method is recommended, for the ease it lends in terms of both installation and upgrade tasks. The more manual DEB package approach, however, is useful in certain situations, such as installing Docker on air-gapped system that have no access to the Internet.

File: ee/docker-ee/ubuntu.md [hunk 8]
```diff
@@ -93,6 +91,8 @@ and update Docker Engine - Enterprise from the repository.
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Edit the text following the **Install using the repository** subheading to read: 
 
Naturally, to install Docker Engine - Enterprise on a new host machine using the Docker repository you must first set the repository up on the machine.

File: ee/docker-ee/ubuntu.md [hunk 9]
```diff
@@ -93,6 +91,8 @@ and update Docker Engine - Enterprise from the repository.
 
 #### Set up the repository
 
+To set up the repository:
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: I suggest deleting this redundant lead-in sentence.

File: ee/docker-ee/ubuntu.md [hunk 10]
```diff
@@ -119,19 +119,20 @@ and update Docker Engine - Enterprise from the repository.
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Edit step 3 to read: "Temporarily add a $DOCKER_EE_URL variable into your environment (it persists only up until you log out of the session)."

File: ee/docker-ee/ubuntu.md [hunk 11]
```diff
@@ -119,19 +119,20 @@ and update Docker Engine - Enterprise from the repository.
...
-   >   Enterprise Edition 2.0 (Docker Engine, UCP, and DTR).
-
     ```bash
     $ DOCKER_EE_VERSION={{ site.docker_ee_version }}
     ```
-
+    > **Note** 
+    > 
+    > If you need to run Docker Enterprise 2.0, please see the
+    > following instructions:
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: I suggest editing the sentence to read: "If you need to run Docker Enterprise 2.0, refer to the following:"

File: ee/docker-ee/ubuntu.md [hunk 12]
```diff
@@ -153,22 +154,23 @@ and update Docker Engine - Enterprise from the repository.
...
     ```
-
+    > **Note** 
+    > 
+    > The included `lsb_release -cs` sub command returns the name of your
+    > Ubuntu distribution, for example, `xenial`.
+    
 #### Install Docker Engine - Enterprise
 
+To install Docker Engine - Enterprise:
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: I suggest deleting this redundant lead-in sentence.

File: ee/docker-ee/ubuntu.md [hunk 13]
```diff
@@ -235,7 +239,7 @@ configuration steps.
 
 #### Upgrade Docker Engine - Enterprise
 
-To upgrade Docker Engine - Enterprise, follow the steps below:
+To upgrade Docker Engine - Enterprise:
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: I suggest deleting this redundant lead-in sentence.

File: ee/docker-ee/ubuntu.md [hunk 14]
```diff
@@ -307,14 +309,16 @@ new file.
 
 ## Uninstall Docker Engine - Enterprise
 
+To uninstall Docker Engine - Enterprise:
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: I suggest deleting this redundant lead-in sentence.

**Comment by traci-morrison** [comment]
> See inline comments.

Changes made. Can you please review and approve? Thanks.

File: ee/docker-ee/ubuntu.md [hunk 15]
```diff
@@ -23,73 +23,58 @@ Docker Engine - Community users should go to
 [Get Docker Engine - Community for Ubuntu](/install/linux/docker-ce/ubuntu.md)
 **instead of this topic**.
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: In my opinion, this line should be the first line of content on the page (following the Docker Enterprise blurb, that is). If possible, it should also render as Important (if that is an option..not quite a Warning, but stronger than just a Note).

Also, the line that follows should be edited to read: 

"To get started with Docker Engine - Enterprise on Ubuntu, confirm that the prerequisites are met and then install Docker."

File: ee/docker-ee/ubuntu.md [hunk 16]
```diff
@@ -23,73 +23,58 @@ Docker Engine - Community users should go to
...
 
-To install Docker Engine - Enterprise, you need to know the Docker Engine -
-Enterprise repository URL associated with your trial or subscription. These
+To install Docker Engine - Enterprise, you need to know the Docker Engine - Enterprise repository URL associated with your trial or subscription. These
 instructions work for Docker on Ubuntu and for Docker on Linux, which includes
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Change the first sentence to read:

"The product URL associated with your trial or subscription is required to install Docker Engine - Enterprise.

File: ee/docker-ee/ubuntu.md [hunk 17]
```diff
@@ -23,73 +23,58 @@ Docker Engine - Community users should go to
 [Get Docker Engine - Community for Ubuntu](/install/linux/docker-ce/ubuntu.md)
 **instead of this topic**.
 
-To install Docker Engine - Enterprise, you need to know the Docker Engine -
-Enterprise repository URL associated with your trial or subscription. These
+To install Docker Engine - Enterprise, you need to know the Docker Engine - Enterprise repository URL associated with your trial or subscription. These
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Change the first sentence to read:

"The product URL associated with your trial or subscription is required to install Docker Engine - Enterprise."

File: ee/docker-ee/ubuntu.md [hunk 18]
```diff
@@ -23,73 +23,58 @@ Docker Engine - Community users should go to
...
 instructions work for Docker on Ubuntu and for Docker on Linux, which includes
 access to Docker Engine - Enterprise for all Linux distributions. To get this
 information, do the following:
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Edit the paragraph to read: "Change the first sentence to read:

"The product URL associated with your trial or subscription is required to install Docker Engine - Enterprise, which can be attained using the following procedure (note that these instructions apply to Docker on Ubuntu and for Docker on Linux, which includes access to Docker Engine - Enterprise for all Linux distributions):"

File: ee/docker-ee/ubuntu.md [hunk 19]
```diff
@@ -23,73 +23,58 @@ Docker Engine - Community users should go to
...
+>
+> The contents of `/var/lib/docker/`, including images, containers, volumes, and networks, are preserved. The Docker Engine - Enterprise package is now called `docker-ee`.
 
 #### Extra steps for aufs
 
 For Ubuntu 16.04 and higher, the Linux kernel includes support for overlay2,
-and Docker Engine - Enterprise uses it as the default storage driver. If you
-need to use `aufs` instead, you need to configure it manually.
-See [aufs](/engine/userguide/storagedriver/aufs-driver.md).
+and Docker Engine - Enterprise uses it as the default storage driver. If you need to use aufs instead, be aware that it must be manually configured (refer to [aufs](/engine/userguide/storagedriver/aufs-driver.md)).
```
Comment (KoryKessel-Docker) [inline ee/docker-ee/ubuntu.md]: Instead of "See aufs." at the end, can we just make the reference to it in the sentence a link?

**Comment by traci-morrison** [comment]
> Some inline comments, and also reiterating a few from a Slack conversation we had:
> 
> Between Steps 5 and 6 (of **Installing Docker Engine - Enterprise** | **Install using the repository** |**SET UP THE REPOSITORY**)there looks to be a Step 5.5 (begins with "Verify that you now..."). If you agree, then that should be step 6 and 6 should be 7 and so on...
> 
> Also...
> 
> Under Step 2 of **Install Docker Engine - Enterprise**, I think the sentence that hangs under the Step 2 sentence that leads to the command is unnecessary. The step sentence is enough of a lead-in (though you might want to add "and containerd" to the step sentence so that it reads: "Install the latest version of Docker Engine - Enterprise and containerd, or go to the next step to install a specific version. Any existing installation of Docker is replaced.").
> 
> Finally...
> 
> Under Step 3 of **Install Docker Engine - Enterprise**, I cannot figure out what the purpose of these two sentences is: "This output is truncated. List the available versions."

The sentences user Step 3 are explained after the command output.

**Comment by traci-morrison** [comment]
@KoryKessel-Docker Changes incorporated. Can you review and approve so that I can merge this today? Thanks.

File: ee/docker-ee/ubuntu.md [hunk 20]
```diff
@@ -13,83 +13,62 @@ toc_max: 4
...
 ### OS requirements
 
 To learn more about software requirements and supported storage drivers,
 check the [compatibility matrix](https://success.docker.com/article/compatibility-matrix).
 
-> **Note:**
-> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.06.xx only.
+> **Note**
+> 
+> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.04.xx only.
```
Comment (thaJeztah) [inline ee/docker-ee/ubuntu.md]: Was changing `17.06` to `17.04` intentional? I don't think we ever shipped a 17.04 enterprise engine

Comment (traci-morrison) [reply ee/docker-ee/ubuntu.md]: Fixed

**Comment by traci-morrison** [comment]
@thaJeztah The version number is now fixed. Can you review and approve so that I can merge this today? Thanks.

**Pull request merged**